Hyper/Controllers/HudController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Common;
```

Comment:
Not sure if this class makes sense. We could just render hud elements in other controllers